console/package.json

```
@@ -79,6 +79,7 @@
     "next-themes": "^0.4.6",
     "orval": "^8.9.1",
     "prettier": "^3.6.2",
+    "qrcode.react": "^4.2.0",
```

Biome (2.5.3)

[error] The key qrcode.react was already declared.

(lint/suspicious/noDuplicateObjectKeys)